Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -144,6 +164,13 @@ public class PortfolioStore: ObservableObject {
           history: priceHistories[priceId] ?? []
         )
       }
+      userVisibleNFTs = nftAssets.map { token in
+        NFTAssetViewModel(
+          token: token,
+          balance: Int(balances[token.assetBalanceId.lowercased()] ?? 0),
+          imageUrl: nil // TODO: fetch image url from erc721Metadata / erc1155Metadata / solana metadata
```

Comment:
If we have a issue filed already maybe we can ref it here

Reply:
Mentioned issue number in [d83aa86](https://github.com/brave/brave-ios/pull/6380/commits/d83aa86134c9d71543df6dbe78e741aff58e92af)